app/services/users.js

```diff
@@ -0,0 +1,49 @@
+const { users } = require('../models'),
+  logger = require('../logger'),
+  errors = require('../errors'),
+  bcrypt = require('bcryptjs');
+const saltRounds = 10;
+
+const haveAllProps = user => user.firstName && user.lastName && user.email && user.password;
```

Comment:
What do you think about express-validator?

Reply:
I think that is great! I will implement it in the next PR (middlewares)